FWCore/ParameterSet/python/Config.py

```diff
@@ -1119,17 +1119,16 @@ def __call__(self,obj):
     for k in self.__args.iterkeys():
         if hasattr(obj,k):
             params[k] = getattr(obj,k)
-        else:
-            params[k] = self.__args[k]
```

Comment:
I'm not 100 % sure if this is a correct fix. I first started to patch `Mixins._modifyParametersFromDict()` to raise an exception if `params[key]` does not have `setValue` attribute. But after staring the code a bit I started to wonder why `self.__args` is copied to `params` here when `Mixins._modifyParametersFromDict()` does effectively same thing.

Anyway, unit tests pass and (subset of) limited matrix workflows give exactly the same config dumps.